core/templates/pages/volunteer-page/volunteer-page.component.ts

```diff
@@ -92,7 +94,9 @@ export class VolunteerPageComponent implements OnInit, OnDestroy {
         this.setPageTitle();
       })
     );
-    this.mapImgPath = '/volunteer/map.png';
+    this.bannerImgPath = '/volunteer/banner.png';
```

Comment:
@hrshkshri Here we are serving png images, maybe we can try using webp instead?

Reply:
![Screenshot 2023-11-05 at 1 54 20 PM](https://github.com/oppia/oppia/assets/108923011/20a87b6a-59d1-413c-8526-23c7cb876e5e)

Hey, @JayVivarekar I tried this already, it failed the frontend karma test.

Reply:
@hrshkshri Which specific test failed after this change? Could you provide the logs?

Reply:
Hi @hrshkshri, could you please answer Jay's question above? Did replacing the types help with the "serve images in next-gen format" lighthouse check failure?

Reply:
I took a look at the commit that was mentioned. It seems like the lighthouse tests passed on that one (https://github.com/oppia/oppia/actions/runs/6711110272)

Can the frontend test failure be corrected by fixing the corresponding unit test (https://github.com/oppia/oppia/actions/runs/6711110268/job/18242660754#step:9:10514)?

Reply:
> I took a look at the commit that was mentioned. It seems like the lighthouse tests passed on that one (https://github.com/oppia/oppia/actions/runs/6711110272)
> 
> Can the frontend test failure be corrected by fixing the corresponding unit test (https://github.com/oppia/oppia/actions/runs/6711110268/job/18242660754#step:9:10514)?

Ah Thanks @kevintab95, Therefore I'll be working on this